packages/server-core/src/projects/project/project-helper.ts

```diff
@@ -1170,7 +1170,7 @@ export const createOrUpdateProjectUpdateJob = async (app: Application, projectNa
   if (k8BatchClient) {
     try {
       await k8BatchClient.patchNamespacedCronJob(
```

Comment:
patch/deleteNamespacedronJob are functions of the k8s SDK, not ones we wrote, so we can't change their behavior. But you're right about making the trimming a single function, which I did.

Reply:
Slicing it to 52 characters is from [k8's documentation on cronjob names](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#:~:text=Even%20when%20the%20name%20is,no%20more%20than%2063%20characters.). It's technically only a limit for cronjob names, not job names, but I figured it couldn't hurt to just use that everywhere. 

The names are mostly a formality, so that someone looking in Lens can see what the job was doing. If we crop a few letters off a project job name unnecessarily, it's not going to affect anything.